editoast/src/modelsv2/scenario.rs

```diff
@@ -0,0 +1,173 @@
+use crate::error::Result;
+use crate::models::List;
+use crate::modelsv2::{Model, Ordering, Row, Tags};
+use crate::views::pagination::{Paginate, PaginatedResponse};
+use async_trait::async_trait;
+use chrono::NaiveDateTime;
+use diesel::sql_query;
+use diesel::sql_types::{BigInt, Text};
+use diesel::{ExpressionMethods, QueryDsl};
+use diesel_async::{AsyncPgConnection as PgConnection, RunQueryDsl};
+use editoast_derive::ModelV2;
+use serde_derive::{Deserialize, Serialize};
+use utoipa::ToSchema;
+
+#[derive(Debug, Clone, ModelV2, Deserialize, Serialize, ToSchema)]
+#[schema(as = ScenarioV2)]
+#[model(table = crate::tables::scenario_v2)]
+pub struct Scenario {
+    pub id: i64,
+    pub infra_id: i64,
+    pub name: String,
+    pub description: String,
+    pub creation_date: NaiveDateTime,
+    pub last_modification: NaiveDateTime,
+    #[model(remote = "Vec<Option<String>>")]
+    pub tags: Tags,
+    pub timetable_id: i64,
+    pub study_id: i64,
+}
+
+#[derive(Debug, Clone, QueryableByName, Deserialize, Serialize, ToSchema)]
+#[schema(as = ScenarioWithDetailsV2)]
+#[diesel(table_name = scenario_v2)]
+pub struct ScenarioWithDetails {
+    #[diesel(embed)]
+    pub scenario: Scenario,
```

Comment:
@SharglutDev spotted a problem in the openapi: you should add `#[schema(value_type = ScenarioV2)]` in order not to reference the old scenario schema